nss-day-cohort-60/national-parks-api
# Users can now post blogs (& Other housekeeping changes)

**Changes:**

> - Defined a SQL helper function for creating new resources in the database
> - Edited the create_blogs function to use the helper function
> - Edited the SQL query for the creation of the blogs table so that photo_id is removed (Later discussion on deleted and recreating our databases is in order. For now, the photo_id exists in the create_blogs function and is required in the request body for those functions to work, but that can be edited after the database discussion is had.)

**How to test:**
1. Git fetch hp-post-blogs
2. Run the python debugger
3. Open Postman
4. Send a POST request to http://localhost:8088/blogs
5. Send a GET request to http://localhost:8088/blogs
6. Check that your POST body is represented by a dictionary there
